tests/pytest_runner.py

```diff
@@ -1,17 +1,29 @@
+import glob
 import os
 
 import pytest
 
-from yt.config import ytcfg
-
 if __name__ == "__main__":
     os.environ["OMP_NUM_THREADS"] = "1"
     pytest_args = [
-        f"--local-dir={os.path.join(ytcfg.get('yt', 'test_data_dir'), 'answers')}",
-        "-c=pytest_answer.ini",
-        "--junitxml=unittests.xml",
-        "--answer-big-data",
-        f"-n {int(os.environ.get('NUM_WORKERS', 6))}",
+        "-s",
+        "-v",
+        "-rsfE",
+        "--with-answer-testing",
+        "-m answer_test",
+        f"-n {int(os.environ.get('NUM_WORKERS', 1))}",
```

Comment:
I don't think the "int" conversion is necessary here ? I mean if it's set to a float, it shouldn't be silently passed down as an int in disguise.

Reply:
```
xarth@shakuras ~ $ export BLAH=1
xarth@shakuras ~ $ python -c 'import os; print(type(os.environ.get("BLAH")))'
<class 'str'>
```

Reply:
🤦🏻 I forgot about that. Disregard !